src/h5web/toolbar/controls/Btn.tsx

```diff
@@ -0,0 +1,47 @@
+import type { AriaAttributes } from 'react';
+import type { IconType } from 'react-icons';
+import styles from '../Toolbar.module.css';
+
+interface Props extends AriaAttributes {
+  label: string;
+  icon?: IconType;
+  iconOnly?: boolean;
+  small?: boolean;
+  raised?: boolean;
+  onClick: () => void;
+  disabled?: boolean;
+}
+
+function Btn(props: Props) {
```

Comment:
This is unrelated (see commits) -- I'm just abstracting a `Btn` component out of `ToggleBtn` and I expose it to @h5web/lib.